### DISCUSSION TITLE
Is there a documented way to host wordpress on subpath ?

### DISCUSSION THREAD

**Discussion by bhanuc** [EasyEngine/easyengine#1609] [Support Requests]
Hi, I need to install wordpress on a subpath  for ex- https://domain.com/subpath/ . 
Is there a documented way to do the same in ee. I tried searching on here and older forums but could not find anything relevant.

**Comment by Alicecommu** [comment]
There are no unique necessities to introduce WordPress in a subdirectory. In the event that you as of now have a WordPress site in the root space name, you are a great idea to go. All top WordPress facilitating organizations make it exceptionally simple to introduce different WordPress sites utilizing the equivalent facilitating account.
